src/views/Manage.js

```diff
@@ -0,0 +1,95 @@
+import ReactDOM from 'react-dom'
+import React from 'react'
+import CardPreview from '../components/CardPreview'
+import CollectionPreview from '../components/CollectionPreview'
+import { collections, getCards } from '../DummyData'
+
+export const Tabbar = ({children, activeTabIndex}) => {
+    let tabs = children;
+
+    return (
+        <nav className="tabbar">
+            <ul className="tab-links">
+                {children}
+            </ul>
+            <hr/>
+        </nav>
+    )
+}
+
+export const Tab = ({index, activeTabIndex, label, onClick}) => {
+
+    let className = index == activeTabIndex ? "tab-active" : "tab";
```

Comment:
I don't think you need both `index` and `activeTabIndex` here.
I'd advise you to replace them both with one `active` prop that will define button style, while `index` will be used in parent component when onClick handler is constructed. Or you may consider keeping `index` prop here for some optimizations, but `activeTabIndex` should definitely go. Reason: outer component should define this component's appearance.